test/system_test.py

```diff
@@ -79,6 +80,29 @@ def two_zip_files_present() -> bool:
             progress_string_fn=zip_file_list,
         )
 
+        assert zip_files
+        for z in zip_files:
+            unzip_dir = tmp_path_factory.mktemp("unzip_dir", numbered=True)
+            self._check_dcm_tags_from_zip(z, unzip_dir)
+
+    def _check_dcm_tags_from_zip(self, zip_path, unzip_dir) -> None:
+        """
+        Check that private tag has survived anonymisation with the correct value.
+        Where are we testing patient name etc?
```

Comment:
Who is this question addressed to? If it's something that still needs to be implemented, maybe create new issue?

Reply:
Yes, sorry, it seemed like a natural place to check all the DICOM metadata, and I don't think we do this anywhere else, so I wrote it down. It should go into an issue since it's not related to the project name feature.